web/src/App.tsx

```diff
@@ -15,28 +15,13 @@
  */
 
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import {BrowserRouter as Router} from 'react-router-dom';
+import Routes from './Routes';
```

Comment:
sort imports?

Reply:
@ifhuang How do you want them sorted? I can add linter rules

Reply:
Right now I'm doing Builtins -> 3rd party deps -> project-level imports, should I just add a newline between each section?

Reply:
Actually I think I'll just deal with all imports ordering in another PR 😃

Reply:
sounds good, and inside each section sort alphabetically